### DISCUSSION TITLE
Evapotranspiration is being underestimated?

### DISCUSSION THREAD

**Discussion by mparedes96** [CH-Earth/summa#518] [General]
Hello everyone.

I'm running SUMMA v3.0 on a mountain basin. I have started to notice that the ET has very low values on the catchment. I have modified the stomatal resistance and soil stress decisions in the decision file, but it does not change much.

What could I do to solve this?

PD: The simulated evapotranspiration is close to 130 [mm/year], while the MODIS Terra product estimates 600 [mm/year].

**Comment by wknoben** [comment]
Thanks for getting in touch!

This is a complicated question because there are many possible causes for this. Just tossing a few out there:
- Issues with the input forcing data (wrong units, uncertainties, missing values)
- Issues with the input geospatial data (incorrect or inappropriate vegetation or soil class)
- Issues with the model parametrization (i.e. evaporation equations being inappropriate for your basin) or the parameter values
- Issues with the analysis code

If you can show some preliminary analysis that suggests that what we're seeing is an issue with the source code we might be able to help further, but this sounds like it could be more of a science question instead. Happy to share some ideas about what to look into but we're going to need a bit more info about your setup than we currently have.
